test/test.js

```diff
@@ -1735,23 +1804,23 @@ describe('comprehensions', () => {
 `for food in ['toast', 'cheese', 'wine']
   eat food`;
     const expected =
-`for (let food of ["toast", "cheese", "wine"]) {
+`for (var food of ["toast", "cheese", "wine"]) {
```

Comment:
All of these loops now use `var` instead of `let`.  Don't know what changed in mainline to cause this.